src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue

```diff
@@ -29,6 +29,7 @@
 				type="button"
 				:aria-expanded="open ? 'true' : 'false'"
 				aria-controls="app-settings__content"
+				:aria-label="inputAriaLabel"
 				@click="toggleMenu">
 				<Cog class="settings-button__icon" :size="20" />
 				<span class="settings-button__label">{{ name }}</span>
```

Comment:
Note: this button has a text value already in `span.settings-button__label {{ name }}`.

Element should not have `aria-label` and direct text at the same time, otherwise it has 2 different sources of label.